frontend/lib/src/components/widgets/DataFrame/columns/DateTimeColumn.ts

```diff
@@ -117,12 +117,12 @@ function BaseDateTimeColumn(
 
   let minDate: Date | undefined = undefined
   if (notNullOrUndefined(parameters.min_value)) {
-    minDate = toSafeDate(parameters.min_value) || undefined
+    minDate = toSafeDate(parameters.min_value.slice(0, 19)) || undefined
```

Comment:
I think this would only work as expected if the min/max dates have the same timezone as all the other values in the column, but that is not a given. I believe the solution here might be to adjust max/min by the calculated timezone offset when creating the individual cells:

```typescript
      let adjustedMinDate: Date | undefined
      if (minDate && timezoneOffset) {
        adjustedMinDate = new Date(
          // Convert timezone offset to milliseconds
          minDate.getTime() + timezoneOffset * 60 * 1000
        )
      }

      let adjustedMaxDate: Date | undefined
      if (maxDate && timezoneOffset) {
        adjustedMaxDate = new Date(
          // Convert timezone offset to milliseconds
          maxDate.getTime() + timezoneOffset * 60 * 1000
        )
      }

      return {
        ...cellTemplate,
        copyData,
        isMissingValue: isNullOrUndefined(cellData),
        data: {
          ...cellTemplate.data,
          date: cellData,
          displayDate,
          timezoneOffset,
          min: adjustedMinDate,
          max: adjustedMaxDate,
        },
      } as DatePickerType
```

But I think this is more of an issue with the date-time-picker of glide data grid. We need to apply the same timezone adjustment that is applied to `date` also to min and max. 

https://github.com/glideapps/glide-data-grid/blob/5983dcabd2fb55b675009813709752008da6d424/packages/cells/src/cells/date-picker-cell.tsx#L75-L85

I will work on a few glide-data-grid aspects a bit later the quarter and can try to fix.